src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "${ModID}:FireVial": {
+    "NAME": "Fire Vial",
+    "DESCRIPTION": "Deal !D! damage. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Deal !D! damage. NL stslib:Exhaustive 2."
+  },
+  "${ModID}:ExplosiveVial": {
+    "NAME": "Explosive Vial",
+    "DESCRIPTION": "Deal !D! damage to ALL enemies. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Deal !D! damage to ALL enemies. NL stslib:Exhaustive 2."
+  },
+  "${ModID}:BlockVial": {
+    "NAME": "Block Vial",
+    "DESCRIPTION": "Gain !B! Block. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Gain !B! Block. NL stslib:Exhaustive 2."
+  },
+  "${ModID}:DynamicVial": {
+    "NAME": "Dynamic Vial",
+    "DESCRIPTION": "Choose: Gain [E] [E] or Draw 3 cards. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Choose: Gain [E] [E] or Draw 3 cards. NL stslib:Exhaustive 2.",
+    "EXTENDED_DESCRIPTION": ["Gain [E] [E] .", "Draw 3 cards."]
+  },
+  "${ModID}:MetallicVial": {
+    "NAME": "Metallic Vial",
+    "DESCRIPTION": "Choose: Gain 4 Plated Armor, Gain 3 Thorns, or Gain 2 Artifact. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Choose: Gain 4 Plated Armor, Gain 3 Thorns, or Gain 2 Artifact. NL stslib:Exhaustive 2.",
+    "EXTENDED_DESCRIPTION": ["Gain 4 Plated Armor.", "Gain 3 Thorns.","Gain 2 Artifact."]
+  },
+  "${ModID}:QuickBrew": {
+    "NAME": "Quick Brew",
+    "DESCRIPTION": "Choose: Gain 2 Strength, or Gain 3 Focus.",
+    "UPGRADE_DESCRIPTION": "Choose: Gain 2 Strength, Gain 2 Dexterity, or Gain 3 Focus.",
+    "EXTENDED_DESCRIPTION": ["Gain 2 Strength.", "Gain 2 Dexterity.","Gain 3 Focus."]
+  },
+  "${ModID}:ToxicVial": {
+    "NAME": "Toxic Vial",
+    "DESCRIPTION": "Choose: Apply 3 Weak, or Apply 8 Poison. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Choose: Apply 3 Vulnerable, Apply 3 Weak, or Apply 8 Poison.",
+    "EXTENDED_DESCRIPTION": ["Apply 3 Vulnerable.", "Apply 3 Weak.","Apply 8 Poison."]
+  },
+  "${ModID}:AttackVial": {
+    "NAME": "Attack Vial",
+    "DESCRIPTION": "Choose 1 of 3 Attacks to gain. It costs 0 until played. NL Exhaust.",
```

Comment:
To add into your hand?

Reply:
I use Gain elsewhere (ghost claws in clawpack) to condense a full line's worth of characters down to four for better line spacing.  If needed we can change it, but I think 'gain' is a sensible shortening.  Obtain is the word the game prefers for permanent effects.